deepchecks/datasets/classification/breast_cancer.py

```diff
@@ -242,7 +244,16 @@ def load_fitted_model():
         model (Joblib model) the model/pipeline that was trained on the iris dataset.
 
     """
-    with urlopen(_MODEL_URL) as f:
-        model = joblib.load(f)
-
+    if sklearn.__version__ == _MODEL_VERSION:
+        with urlopen(_MODEL_URL) as f:
+            model = joblib.load(f)
+    else:
+        model = _build_model()
```

Comment:
Maybe add a logger.info that the sklearn version is different and we are building a model